### ISSUE TITLE
[BUG] No permission to open the application (Mac OS)

### ISSUE THREAD

**Issue by LockStep-Github** [Lightcord/Lightcord#128]
**Cannot open Lightcord after Downloading**
I downloaded Lightcord on my Mac, and I am met with an error that says this: `You do not have permission to open the application “Lightcord”.`

**To Reproduce**
1. Download Lightcord Mac Zip file
2. Extract to Desktop
3. Double Click on Lightcord

**Expected behavior**
I expected Lightcord to open normally and perhaps install updates.

**Screenshots**
[Error Image](https://monosnap.com/direct/rk940zkUry2NV3vssepIf8h2iyn7b0)
[Proof of Lightcord Download](https://monosnap.com/direct/egjPPfiRfLHnuTjCegjR12vuQSeM8T)

**Desktop:**
 - OS: macOS Big Sur
 - Version 11.0.1

**Additional comments**
I've snooped around online and on Stack Overflow, and none of those suggested fixes works. This points to a problem on the side of Lightcord, i.e. the fact that Big Sur is so new may play a part and is thus incompatible.

My Mac is not a work computer, it is my personal one. I am the only user on it and I have full admin permissions yet I receive this error.

**Comment by GermanBread** [comment]
Run it as root

**Comment by LockStep-Github** [comment]
> Try running LC as root. 

This doesn't work.

> Alternatively, try clicking on the ? 

This just brings me to an Apple Help page that is ironically of no help.

> Edit: Scratch that,  your app might be lacking execute permissions

Is this something I can fix on my end, or will I have to wait for a Lightcord update?

**Comment by GermanBread** [comment]
> > Try running LC as root.
> 
> This doesn't work.
> 
> > Alternatively, try clicking on the ?
> 
> This just brings me to an Apple Help page that is ironically of no help.
> 
> > Edit: Scratch that,  your app might be lacking execute permissions
> 
> Is this something I can fix on my end, or will I have to wait for a Lightcord update?

You need to do this on your end, UNIX and UNIX-alike OSes, unlike NT based ones, require excecutables to be marked as executable.
TL;DR: 
https://discussions.apple.com/thread/3715259
https://stackoverflow.com/questions/15080835/running-an-executable-in-mac-terminal

**Comment by LockStep-Github** [comment]
I tried all of the solutions linked above, none seem to work. 

>https://discussions.apple.com/thread/3715259

This is a thread about running .exe files on a Mac. I downloaded a Zip, extracted it, and now I have a .app file, so that thread was not much use.

>https://stackoverflow.com/questions/15080835/running-an-executable-in-mac-terminal

These commands did not work either. Not too sure what's going on.

**Comment by typicalMoves** [comment]
For me, Lightcord just hangs after logging in and will just endlessly eat my CPU until I force quit the app. The spinner spins once in one foul, perfect rotation. Prior to logging in, it's fine.
